README.md

```diff
@@ -4,15 +4,17 @@ Example push notification server, written in Golang
 
 ## Project status
 
-This repo is very much a WIP, and some important features are not yet completed.
+This project is in Developer Preview. Software in this phase does is not formally supported, will change without warning, and may not be backwards compatible. We do NOT recommend using Developer Preview software in production applications.
```

Comment:
Could we also add our fancy new badge for consistency? ![Status](https://img.shields.io/badge/Project_Status-Developer_Preview-yellow)

Reply:
Ooh. So fancy

Reply:
Thank you @elisealix22 and @neekolas - reading this beauty TODAY! <3